android/app/src/main/java/com/cBreez/client/BreezFCMService.kt

```diff
@@ -19,25 +20,31 @@ class BreezFcmService : FirebaseMessagingService() {
 
     override fun onMessageReceived(remoteMessage: RemoteMessage) {
         super.onMessageReceived(remoteMessage)
+        configureLogger()
```

Comment:
This creates a logger and configure it with every notification. It is better IMO to configure it once and set it statically.